packages/fxa-shared/payments/stripe.ts

```diff
@@ -474,7 +477,7 @@ export abstract class StripeHelper {
    */
   async expandResource<T>(
     resource: string | T,
-    resourceType: typeof VALID_RESOURCE_TYPES[number],
+    resourceType: (typeof VALID_RESOURCE_TYPES)[number],
```

Comment:
I checked with @julianpoy and this should be reverted back. Auto-lint fail.

Example from Julian:
```
VALID_RESOURCE_TYPES = [1, 'asdf'];

resourceType = typeof VALID_RESOURCE_TYPES[0]; // resourceType will be 'number'
resourceType = (typeof VALID_RESOURCE_TYPES)[0]; // resourceType will be 'o'
```

Reply:
fixed 👍 good catch